pkg/util/kubernetes/hostinfo/tags.go

```diff
@@ -48,6 +53,15 @@ func GetTags(ctx context.Context) (tags []string, err error) {
 		}
 	}
 
+	ku, err := kubeUtilGet()
+	if err != nil {
+		return
+	}
+	nodeName, err := ku.GetNodename(ctx)
```

Comment:
can we get the node name thanks to the nodeInfo?  [NewNodeInfo](https://github.com/DataDog/datadog-agent/pull/17982/files#diff-47884fff715f0ed85142c79f37b41d3f932734d5cf5c565f8ae484af67ed01eaR30)